lib/context/engine-context.js

```diff
@@ -44,7 +44,8 @@ class EngineContext {
             pipeline: true,
             startWorkerDuration: Utils.durationSec(this[INTERNAL].processingStartTime, this[INTERNAL].workerStartTime),
             gatewayToProcessDuration: Utils.durationSec(params.times.gateway, params.times.process),
-            processToCoreDuration: Utils.durationSec(params.times.process, params.times.core)
+            processToCoreDuration: Utils.durationSec(params.times.process, params.times.core),
+            renditionCount: 1
```

Comment:
Maybe already write the code doing the count, then?

Reply:
+1 if its not big in scope. if not add a comment so that we have the context.